### DISCUSSION TITLE
Update a custom field based on another custom field's value

### DISCUSSION THREAD

**Discussion by plogramer** [sadieblue/agilityblue.workspace-scripts#6] [Q&A] [answered]
I have two custom fields in Billing Entry object.

1. DELETE (Yes / No)
2. Billable (Yes / No)

When user sets "DELETE" to true, I want to set "Billable" as false before saving the change. 

![image](https://github.com/user-attachments/assets/f6a958ce-0716-4d3b-8f23-6f19eb098fab)

**Comment by bmward80** [comment, marked as answer]
Hi, @plogramer. If you check the `Value` field, you will find that it will contain a `ValueAsBoolean` property you can check to perform your comparison. When you convert the object to json to inspect it, make sure to add `-Depth 10` or something higher than PowerShell's default of 2 (this is why you are seeing "AgilityBlue.ViewModels.Workspace.ObjectFieldValueViewModel" instead of the json value of it).

Check out the [https://help.agilityblue.com/docs/scripting-tutorials-system-objects-with-custom-fields-intro](https://help.agilityblue.com/docs/scripting-tutorials-system-objects-with-custom-fields-intro) article on information and examples on how to build out the structure of the custom field you want to provide where the Value is showing `null`.

I hope this helps, and let me know if there's anything else,
Brandon

Edit: I forgot to add that you can also see how to specifically provide a boolean value in this article, also. Good examples there: [https://help.agilityblue.com/docs/scripting-tutorials-custom-fields-deep-dive#data-type-yes-or-no-choice](https://help.agilityblue.com/docs/scripting-tutorials-custom-fields-deep-dive#data-type-yes-or-no-choice)
